frameworks/Go/go-raw/src/hello/hello.go

```diff
@@ -34,18 +34,7 @@ type Fortune struct {
 
 // Databases
 const (
-	// Go 1.4's sql.DB has scalability problem when using (explicitly reused) prepared statement.
-	// https://github.com/golang/go/issues/9484
-	//
-	// Using db.Query() instead of stmt.Query() avoid the issue.
-	// But it makes 3 round trips per query: prepare, execute and close.
-	// `interpolateParams=true` enables client side parameter interpolation.
-	// It reduces round trips without prepared statement.
-	//
-	// Before Go 1.5 is released, we can see real power of Go with this benchmark.
-	// After Go 1.5 is released, we can see prepared statement vs interpolation by comparing
-	// this and another lightweight Go framework.
```

Comment:
> there are other differences between frameworks, so it is impossible to explain the difference in numbers by a single factor (prepared vs unprepared statements)

There are many "micro" frameworks in Go.  Performance difference between them are only URL routing.
In case of "multiple queries" benchmark, difference other than DB access is small enough to ignore.

> scalability of prepared statements has been fixed in Go 1.5

Yes. I'm who fixed it.
Have you *really* read the comment?
"After Go 1.5 is released, we can see prepared statement vs interpolation by comparing
 this and another lightweight Go framework."

> a version with prepared statements has the same or better performance with Go 1.5 and mysql

I want to see it on "Framework Benchmarks result" page.

> using prepared statements is more secure and generally should be prefered in real production usage

I implemented "interpolateParams" secure and I use it on real production.
There are some pros in prepared statement:

* Some ORM doesn't support reusing prepared statement.  In such case, prepared statement requires three
roundtrips (prepare, exec, close).
* "SHOW FULL PROCESSLIST" is handy tool for DB admins.  Prepared statement hides real SQL from PROCESSLIST.